roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage
+  lineinfile: dest=/etc/default/grub
+    state=present
+    regexp="^GRUB_CMDLINE_LINUX_DEFAULT=\"(.*)\"$"
+    line="GRUB_CMDLINE_LINUX_DEFAULT=\"\1 transparent_hugepage=never\""
+    backrefs=yes
+  when: ceph_osd.disable_transparent_hugepage
```

Comment:
Actually yeah, there is http://manpages.ubuntu.com/manpages/trusty/man8/hugeadm.8.html which could be used. Not sure if it survives a reboot, but worth testing. Sorry for missing this in earlier reviews :(